contracts/payloads/IGP93/PayloadIGP93.sol

```diff
@@ -0,0 +1,737 @@
+
+import {IFluidVaultFactory} from "../common/interfaces/IFluidVaultFactory.sol";
+import {IFluidDexFactory} from "../common/interfaces/IFluidDexFactory.sol";
+
+import {IFluidDex, IFluidAdminDex, IFluidDexResolver} from "../common/interfaces/IFluidDex.sol";
+
+import {IFluidVault, IFluidVaultT1} from "../common/interfaces/IFluidVault.sol";
+
+import {IFTokenAdmin, ILendingRewards} from "../common/interfaces/IFToken.sol";
+
+import {ISmartLendingAdmin} from "../common/interfaces/ISmartLending.sol";
+import {ISmartLendingFactory} from "../common/interfaces/ISmartLendingFactory.sol";
+import {IFluidLendingFactory} from "../common/interfaces/IFluidLendingFactory.sol";
+
+import {ICodeReader} from "../common/interfaces/ICodeReader.sol";
+
+import {IDSAV2} from "../common/interfaces/IDSA.sol";
+import {IERC20} from "../common/interfaces/IERC20.sol";
+import {IProxy} from "../common/interfaces/IProxy.sol";
+import {PayloadIGPConstants} from "../common/constants.sol";
+import {PayloadIGPHelpers} from "../common/helpers.sol";
+import {PayloadIGPMain} from "../common/main.sol";
+
+contract PayloadIGP93 is PayloadIGPMain {
+    uint256 public constant PROPOSAL_ID = 93;
+
+    function execute() public virtual override {
+        super.execute();
+
+        // Action 1: Collect ETH Revenue and Transfer to Multisig
+        action1();
+
+        // Action 2: Update Borrow Rate Magnifier on LBTC-CBBTC / WBTC Vault
+        action2();
+
+        // Action 3: Adjust Rate curves of WBTC & cbBTC
+        action3();
+
+        // Action 4: Adjust CF, LT, LML for GHO Vaults
+        action4();
+
+        // Action5: Update expand percentage and duration for fTokens
+        action5();
+
+        // Action 6: Reduce limits on unused USDE debt vaults
+        action6();
+
+        // Action 7: Readjust the Max Borrow Limit for LBTC-cbBTC<>cbBTC vault
+        action7();
+
+        // Action 8: Set dust limits for Gold based vaults and DEX
+        action8();
+
+        // Action 9: Transfer $FLUID to Team Multisig for Rewards
+        action9();
+
+        // Action 10: Update borrow caps for cbBTC-wBTC Dex pool and USDC-USDT pool
+        action10();
+
+        // Action 11: Update lower range of weETHs-ETH & lower range and fees of rsETH-ETH
+        action11();
+    }
+
+    function verifyProposal() public view override {}
+
+    function _PROPOSAL_ID() internal view override returns (uint256) {
+        return PROPOSAL_ID;
+    }
+
+    /**
+     * |
+     * |     Proposal Payload Actions      |
+     * |__________________________________
+     */
+
+    // @notice Action 1: Collect ETH Revenue and Transfer to Multisig
+    function action1() internal isActionSkippable(1) {
+        address[] memory tokens = new address[](1);
+
+        tokens[0] = ETH_ADDRESS;
+
+        LIQUIDITY.collectRevenue(tokens);
+
+        uint256[] memory amounts = new uint256[](1);
+
+        amounts[0] = 128 ether; // 129 ETH in reserves - 1
```

Comment:
need to do `address(FLUID_RESERVE).balance - 1;` to include the reserves coming in from collecting